app/containers/message/Components/CollapsibleQuote.test.tsx

```diff
@@ -0,0 +1,91 @@
+import { fireEvent, render, within } from '@testing-library/react-native';
```

Comment:
I'm ok with adding this lib, but we should keep updating stories until we migrate to `@testing-library`, so we have one source of truth.